python/fledge/services/core/api/support.py

```diff
@@ -160,6 +166,9 @@ async def get_syslog_entries(request):
         template = __GET_SYSLOG_CMD_TEMPLATE
         lines = __GET_SYSLOG_TOTAL_MATCHED_LINES
         non_total_template = __GET_SYSLOG_TEMPLATE_WITH_NON_TOTALS
```

Comment:
Local variable 'non_total_template' value is not used anymore. Remove it from here and its reference.

Reply:
I would like to retain it for now while this change is being tested by Ashwin and others. I can maybe comment it for now.